config/database.yml

```diff
@@ -24,8 +24,8 @@ default: &default
 development:
   <<: *default
   database: kata_07_reloj_checador_k7_f22_t3_development
-  username: 
-  password:
+  username: alex
+  password: qw12..
```

Comment:
use `ENV` variables instead of fixed data, check the [dotenv gem](https://github.com/bkeepers/dotenv) 
add a `.env-example` file, keep it always up to date, whit the env vars that you are using in your project